msgs/Range.proto

```diff
@@ -1,10 +1,14 @@
 syntax = "proto2";
 package sensor_msgs.msgs;
+import "quaternion.proto";
 
 message Range
 {
   required int64 time_usec       	= 1;
   required float min_distance      	= 2;
   required float max_distance  	 	= 3;
-  required float current_distance     	= 4;
+  required float current_distance   = 4;
```

Comment:
indentation

Reply:
what style is applied in this repo? Cannot find anything in the readme

Reply:
well for proto files I am not seeing any defined style. But at least you could align the numbers :)

Reply:
For the C++ files, if I am not wrong, it's Google C++ Style. But we are not currently enforcing it.